src/transaction/transaction_group_complete_manager.cpp

```diff
@@ -129,8 +268,10 @@ namespace cubtx
   {
     m_latest_closed_group_state |= GROUP_MVCC_COMPLETED;
 
+#if defined (SERVER_MODE)
```

Comment:
Avoiding notify for SA mode seems a little too pedantic... Is it needed?

Reply:
In SA notify has no sense. But, since group_complete_manager::notify_all already consider the case, we can remove duplicate in methods like group_complete_manager::notify_group_mvcc_complete.